<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_excel_path">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Excel表格：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_path">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>选择</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_load">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>加载</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_sheet">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>40</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_hang_value">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>40</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>36</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="text">
     <string>表单</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>80</y>
      <width>601</width>
      <height>311</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_1">
     <attribute name="title">
      <string> 设置样式</string>
     </attribute>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>61</width>
        <height>151</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_1">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="focusPolicy">
          <enum>Qt::TabFocus</enum>
         </property>
         <property name="text">
          <string>段首</string>
         </property>
         <property name="textFormat">
          <enum>Qt::AutoText</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="focusPolicy">
          <enum>Qt::TabFocus</enum>
         </property>
         <property name="text">
          <string>每行头</string>
         </property>
         <property name="textFormat">
          <enum>Qt::AutoText</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="focusPolicy">
          <enum>Qt::TabFocus</enum>
         </property>
         <property name="text">
          <string>每行尾</string>
         </property>
         <property name="textFormat">
          <enum>Qt::AutoText</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="focusPolicy">
          <enum>Qt::TabFocus</enum>
         </property>
         <property name="text">
          <string>段尾</string>
         </property>
         <property name="textFormat">
          <enum>Qt::AutoText</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTextEdit" name="textEdit_1">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>30</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>70</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_3">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>110</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_4">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>150</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_checkprint">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>240</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>查看样式</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>样式预览与生成log</string>
     </attribute>
     <widget class="QTextBrowser" name="textBrowser_print">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>481</width>
        <height>181</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_start">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>240</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>开始生成</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_print_info">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>状态：</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="labelsheet_log">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>0</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>581</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
